drone/src/agent/engines/docker/util.rs

```diff
@@ -87,6 +87,8 @@ impl ContainerEvent {
         let action = event.action.as_deref()?;
         let actor = event.actor.as_ref()?;
         let name: String = actor.attributes.as_ref()?.get("name")?.to_string();
+        // Some actions add extra metadata after a colon, we strip that out.
```

Comment:
Lots of commands. I think it came up for the first time because in the devcontainer, we share a Docker instance with the one that runs the container, so we get noise from other stuff that Codespaces does with it.

Here's a sample of the log noise:

```
plane-drone       | 2022-11-18T19:12:00.730292Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: top -bn1"
plane-drone       | 2022-11-18T19:12:00.730757Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_start: top -bn1"
plane-drone       | 2022-11-18T19:12:04.578390Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: cat /var/run/sshd.pid"
plane-drone       | 2022-11-18T19:12:04.579193Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_start: cat /var/run/sshd.pid"
plane-drone       | 2022-11-18T19:12:05.956975Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: top -bn1"
plane-drone       | 2022-11-18T19:12:05.957303Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_start: top -bn1"
plane-drone       | 2022-11-18T19:12:11.185387Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: top -bn1"
plane-drone       | 2022-11-18T19:12:11.185896Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_start: top -bn1"
plane-drone       | 2022-11-18T19:12:14.661318Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: cat /var/run/sshd.pid"
plane-drone       | 2022-11-18T19:12:14.661855Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_start: cat /var/run/sshd.pid"
plane-drone       | 2022-11-18T19:12:16.412237Z  INFO plane_drone::agent::engines::docker::util: Unhandled container action. action="exec_create: top -bn1"
```

Reply:
Ahhh interesting